panther_lights/src/animation/battery_animation.py

```diff
@@ -6,16 +6,17 @@
 
 
 class BatteryAnimation(Animation):
-
     ANIMATION_NAME = 'battery_animation'
 
     def __init__(self, *args, **kwargs) -> None:
         super().__init__(*args, **kwargs)
 
         self._anim = np.zeros((self._anim_len, self.num_led))
-        self._h_min = 0.0
-        self._h_max = 120.0
+        self._h_min = 0.0  # red color
+        self._h_max = 0.4  # green color
```

Comment:
```suggestion
        self._h_max = 120.0 / 360.0  # green color
```
For someone familiar with HSV this value is a bit abstract and also actual green would be 0.(3). If we want to keep `hsv_to_rgb` method format I suggest simple division to make it more descriptive.